frontend/sozisel-app/src/components/utils/Spinner/Spinner.scss

```diff
@@ -0,0 +1,85 @@
+@use "../../../styles/color.scss";
+
+.loader {
```

Comment:
Forget about this component and use https://www.npmjs.com/package/react-spinners.